ui/plugin-system/src/components/MultiQueryEditor/QueryEditorContainer.test.tsx

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 The Perses Authors
```

Comment:
does not really matter, but just for the sanity we are in 2025 :)

```suggestion
// Copyright 2025 The Perses Authors
```